calico-vpp-agent/cmd/calico_vpp_dataplane.go

```diff
@@ -93,12 +94,24 @@ func main() {
 	}
 	// Once we have the api connection, we know vpp & vpp-manager are running and the
 	// state is accurately reported. Wait for vpp-manager to finish the config.
-	err = common.WaitForVppManager()
+	common.VppManagerInfo, err = common.WaitForVppManager()
 	if err != nil {
 		log.Fatalf("Vpp Manager not started: %v", err)
 	}
-
 	common.ThePubSub = common.NewPubSub(log.WithFields(logrus.Fields{"component": "pubsub"}))
+	routeWatcher := watchers.NewRouteWatcher()
+
+	Go(routeWatcher.WatchRoutes)
```

Comment:
We could probably move those watchers next to the other ones (a.k.a. instantiation next to `NewNodeWatcher()` and startup next to `Go(nodeWatcher.WatchNodes)`)